config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  root to: "items#index"
+  get 'current_user/index'
```

Comment:
Ditto in relation to my comment in the current_user controller. How is this endpoint necessary for the app?

Reply:
:heavy_check_mark: Removed! 
It was not needed, indeed. :+1: